packages/clay-nav/src/Vertical.tsx

```diff
@@ -46,9 +47,9 @@ interface IItemWithItems extends IItem {
 
 interface IProps {
 	/**
-	 * Label of item that is currently active.
+	 * Label of the button that appears on smaller resolutions to open the vertical navigation.
 	 */
-	activeLabel: string;
+	buttonToggleLabel?: string;
```

Comment:
@kresimir-coko making this change will cause a breaking change since we would be deprecated the `activeLabel` API, so I think we can just keep `activeLabel`.

Reply:
You're right, my bad!